### PR TITLE
disk: fix parameter checks in NewNodeServer

### PR DESCRIPTION
**Pull request by dongjiang1989**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:

fix parameter checks in NewNodeServer

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by mowangdk** [comment]
@dongjiang1989 yep, you could help with the error message & constants name fix. thx~

> I think this is intentional. `MAX_VOLUMES_PERNODE` (0-64) can override the default value from `MaxVolumesPerNode` (15). Just the error message should be fixed.

**Comment by dongjiang1989** [comment]
> @dongjiang1989 yep, you could help with the error message & constants name fix. thx~
> 
> > I think this is intentional. `MAX_VOLUMES_PERNODE` (0-64) can override the default value from `MaxVolumesPerNode` (15). Just the error message should be fixed.

Maybe `DefaultMaxVolumesPerNode` is 15, and `MaxVolumesPerNodeLimit` is 64.

File: pkg/disk/nodeserver.go [hunk 1]
```diff
@@ -118,15 +120,15 @@ type QueryResponse struct {
...
+	var maxVolumesNum int64 = DefaultMaxVolumesPerNode
 	volumeNum := os.Getenv("MAX_VOLUMES_PERNODE")
 	if "" != volumeNum {
 		num, err := strconv.ParseInt(volumeNum, 10, 64)
 		if err != nil {
 			log.Log.Fatalf("NewNodeServer: MAX_VOLUMES_PERNODE must be int64, but get: %s", volumeNum)
 		} else {
-			if num < 0 || num > 64 {
-				log.Log.Errorf("NewNodeServer: MAX_VOLUMES_PERNODE must between 0-15, but get: %s", volumeNum)
+			if num < 0 || num > int64(MaxVolumesPerNodeLimit) {
```
Comment (huww98) [inline pkg/disk/nodeserver.go:130]:
```suggestion
			if num < 0 || num > MaxVolumesPerNodeLimit {
```
consts are untyped, I think we don't need the conversion

File: pkg/disk/nodeserver.go [hunk 2]
```diff
@@ -118,15 +120,15 @@ type QueryResponse struct {
...
 	volumeNum := os.Getenv("MAX_VOLUMES_PERNODE")
 	if "" != volumeNum {
 		num, err := strconv.ParseInt(volumeNum, 10, 64)
 		if err != nil {
 			log.Log.Fatalf("NewNodeServer: MAX_VOLUMES_PERNODE must be int64, but get: %s", volumeNum)
 		} else {
-			if num < 0 || num > maxVolumesNum {
-				log.Log.Errorf("NewNodeServer: MAX_VOLUMES_PERNODE must between 0-%d, but get: %s", maxVolumesNum, volumeNum)
+			if num < 0 || num > MaxVolumesPerNodeLimit {
+				log.Log.Errorf("NewNodeServer: MAX_VOLUMES_PERNODE must between 0-%d, but get: %s", int64(MaxVolumesPerNodeLimit), volumeNum)
```
Comment (huww98) [inline pkg/disk/nodeserver.go:131]:
```suggestion
				log.Log.Errorf("NewNodeServer: MAX_VOLUMES_PERNODE must between 0-%d, but get: %s", MaxVolumesPerNodeLimit, volumeNum)
```
The same to this one.

Comment (dongjiang1989) [reply pkg/disk/nodeserver.go]: Done. Fix nits

**Pull request merged**